{"name": "test_mul[9-6-15]", "status": "failed", "statusDetails": {"message": "AssertionError: Test Failed!!!\nassert 15 == 54", "trace": "self = <Work.test_params.TestParams object at 0x10df37550>, a = 9, b = 6\nexpected = 15\n\n    @allure.description(\"乘法计算\")\n    @pytest.mark.parametrize(\"a,b,expected\", yaml.load(open(\"params.yaml\"))[\"Mul\"])\n    def test_mul(self, a, b, expected):\n        sum = mul(a, b)\n>       assert expected == sum, \"Test Failed!!!\"\nE       AssertionError: Test Failed!!!\nE       assert 15 == 54\n\ntest_params.py:63: AssertionError"}, "description": "乘法计算", "attachments": [{"name": "log", "source": "cdc16c70-329d-43b8-bdb0-c997f85dbef4-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "5da29088-b8f7-4b8d-b4ad-2f27b7fdb80a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "9"}, {"name": "b", "value": "6"}, {"name": "expected", "value": "15"}], "start": 1573546636578, "stop": 1573546636578, "uuid": "90a447a9-5e76-4869-9cc8-b118510dfd93", "historyId": "5147a9713d2dc0e7bb5e2bdfd74bd868", "testCaseId": "6107f4c26291394d677dde20760f05d3", "fullName": "test_params.TestParams#test_mul", "labels": [{"name": "suite", "value": "test_params"}, {"name": "subSuite", "value": "TestParams"}, {"name": "host", "value": "ZTL.local"}, {"name": "thread", "value": "3894-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_params"}]}